cocotb/regression.py

```diff
@@ -180,6 +181,10 @@ def initialise(self):
             for thing in vars(module).values():
                 if hasattr(thing, "im_test"):
                     self._init_test(thing)
+                    have_tests = True
+
+        if not have_tests:
```

Comment:
Could you use `if self._queue` here? Or does that not work?

Reply:
Thing are only added to the queue if they initialize properly and are not skipped. What we want to see is if we find anything marked as a `test`.

Reply:
I'd perhaps argue we want a warning for either case, but I'm happy to call that out of scope.